disease-classification-training/unittests.py

```diff
@@ -21,13 +21,17 @@ def setUp(self) -> None:
         self.disease_classification = train_model(
```

Comment:
syntax error on line 3 of this file: sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))import numpy as np

Reply:
Did you run this with tf.keras.model installed in your local system?